packages/react/__tests__/exports.ts

```diff
@@ -1866,23 +1868,16 @@ describe('@aws-amplify/ui-react', () => {
           "VisuallyHidden",
           "Wrapper",
           "components",
-          "convertStylePropsToStyleObj",
```

Comment:
Removed these util functions, because I don't see any reason for them to be exposed on the public api.

Reply:
thanks!